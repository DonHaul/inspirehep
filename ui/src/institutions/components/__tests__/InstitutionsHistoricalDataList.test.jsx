import React from 'react';
import { render } from '@testing-library/react';
import { fromJS } from 'immutable';
import InstitutionsHistoricalDataList from '../InstitutionsHistoricalDataList';

describe('InstitutionsHistoricalDataList', () => {
  it('renders', () => {
    const historicalData = fromJS(['This is my first note', 'second note']);
    const { asFragment } = render(
      <InstitutionsHistoricalDataList historicalData={historicalData} />
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
