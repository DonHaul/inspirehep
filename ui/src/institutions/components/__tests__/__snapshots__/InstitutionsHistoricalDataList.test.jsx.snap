// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InstitutionsHistoricalDataList renders 1`] = `
<DocumentFragment>
  <div
    class="__InlineList__"
    data-testid="inline-data-list"
  >
    <span
      class=""
    >
      Note: 
    </span>
    <ul>
      <li>
        This is my first note â€¢ 
      </li>
      <li>
        second note
      </li>
    </ul>
  </div>
</DocumentFragment>
`;
